### DISCUSSION TITLE
can't add email to github

### DISCUSSION THREAD

**Discussion by JonLlo** [community/community#84041] [New to GitHub]
Every time I try to add my email to github, I get an error page.
<img width="734" alt="Screen Shot 2024-01-03 at 12 24 22" src="https://github.com/community/community/assets/154521758/5935d63f-9ffe-4eee-8288-8aa4420820fe">

**Comment by willianctti** [comment]
to me too

**Reply by vadskev** [reply]
Hi, i have this problem too

**Comment by fury-05** [comment]
Its a bug, it will be resolved soon try again after some time.

**Comment by Super-Andy-Shen** [comment]
Please bring it back asap

**Comment by AyushB21** [comment]
Actually this is a BUG which I was having for a long time even, its jyst you clear your browser cache and redo, if noy try using Incognito
